frontend/polyfill-mutation-observer.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable */
+
+global.MutationObserver = global.MutationObserver || class {
```

Comment:
We do not need proper polyfill for tests, like [1].

@spadgett , what do you think about location of this file?

[1] https://www.npmjs.com/package/mutation-observer

Reply:
> We do not need proper polyfill for tests, like [1].

In that case, add this as `__mocks__/mutationObserver.ts` and place it **before** `before-tests.js` in Jest `setupFiles` list.

The `before-tests.js` script should contain custom test setup code executed **after** loading all pre-requisites such as DOM API mocks.

Reply:
done